src/mudlet.cpp

```diff
@@ -416,15 +416,6 @@ mudlet::mudlet()
     timerAutologin->start( 1000 );
 
     connect(mpMainStatusBar, SIGNAL(messageChanged(QString)), this, SLOT(slot_statusBarMessageChanged(QString)));
-    // Do something with the QStatusBar just so we "use" it (for 15 seconds)...
```

Comment:
⚠️ Confirmed my suspicions - it needs to be shown at least one for it to automatically show the `statusTips` that we do use - if they are wanted later - will revert but revise text in a suitable manner...